app/pb/auth.proto

```diff
@@ -139,4 +153,14 @@ message DeAuthReq {
   string token = 1;
 }
 
-message DeAuthRes {}
+message ResetPasswordReq {
+  string user = 1;
```

Comment:
Please spell out `username ` to not confuse with user_id.

Reply:
This actually takes any of `username`, `user_id` or `email`, akin to the other functions that user `get_user_by_field`.